.github/workflows/build.yml

```
@@ -249,6 +249,15 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Set up or Update vcpkg
```

```suggestion
      - name: Update vcpkg (TEMPORARY)
```

This manual vcpkg update won't be needed once the `windows-2019` runner image updates vcpkg to include [the fix that was recently merged](https://github.com/microsoft/vcpkg/pull/41574). I don't know how long it will take them to do that - could be a few weeks - but we'll want to remove this build step after that happens.

---

I changed the description